cmd/cleanup-controller/server.go

```diff
@@ -75,7 +76,7 @@ func NewServer(
 	mux.HandlerFunc("GET", config.ReadinessServicePath, handlers.Probe(probes.IsReady))
 	return &server{
 		server: &http.Server{
-			Addr: ":9443",
+			Addr: ":" + internal.CleanupServerPort(),
```

Comment:
FYI @chipzoller - the port can be configurable after this change.

Reply:
Thanks